packages/venia-concept/src/components/MiniCart/miniCart.js

```diff
@@ -95,11 +96,30 @@ class MiniCart extends Component {
                         <Icon name="x" />
                     </Trigger>
                 </div>
-                <div className={classes.body}>{productList}</div>
-                <div className={classes.footer}>
-                    <div className={classes.summary}>{totalsSummary}</div>
-                </div>
-                <Checkout cart={cart} />
+                {cart.details.items && cart.details.items.length ? (
+                    <Fragment>
+                        <div className={classes.body}>{productList}</div>
+                        <div className={classes.footer}>
+                            <div className={classes.summary}>
+                                {totalsSummary}
+                            </div>
+                        </div>
+                        <Checkout cart={cart} />
+                    </Fragment>
+                ) : (
+                    <div className={classes.empty}>
```

Comment:
Thank you for the pull request! Could you please move this into a separate component `EmptyMiniCart`.

Reply:
Thanks, just updated the PR with the change